packages/flutter/test/material/scaffold_test.dart

```diff
@@ -574,6 +574,62 @@ void main() {
       expect(tester.element(find.byKey(testKey)).size, const Size(88.0, 48.0));
       expect(tester.renderObject<RenderBox>(find.byKey(testKey)).localToGlobal(Offset.zero), const Offset(0.0, 0.0));
     });
+
+    WidgetTesterCallback getBodySizeTesterCallback({bool expanded = false, Size size}) => (WidgetTester tester) async {
```

Comment:
I think the overall goal here is to verify the height of the Scaffold's body for the eight configurations possible with: extendedBody, resizeToAvoidBottomInset, and viewInset.bottom = 0 or 100 (keyboard is up or down). A single test, roughly like `body size with resizeToAvoidBottomInset...' with a named parameter buildFrame method would be easier to understand.

```
buildFrame({ extendedBody, resizeToAvoidBottomInset, viewInsetBottom })
```